manticore/platforms/linux.py

```diff
@@ -250,6 +255,11 @@ def _transmit(self, buf):
             self.buffer.insert(0,c)
         return len(buf)
 
+    def sync(self):
+        raise BadFd("Invalid sync() operation on Socket")
```

Comment:
we actually do not want to raise here, we don't want analysis to crash/stop if the program has an fsync, which is why it was just a nop earlier